### ISSUE TITLE
bug in def process_single_file from /infer /wsi.py

### ISSUE THREAD

**Issue by YOOHYOJEONG** [TissueImageAnalytics/cerberus#33]
Hi. 

I found a bug while trying to run_infer_wsi.py .

When using msk_file, the inst_info["box"] is returning the wrong coordinates.

[Here](https://github.com/TissueImageAnalytics/cerberus/blob/5bcecbb071bebd5911250034c94f3568f23f50bb/infer/wsi.py#L816)(816 line at /infer /wsi.py), 
if you add tissue_topleft ([cmin, rmin]) to the inst_info["box"] coordinates, the x and y coordinates are added in reverse.

In other words, you need to add [rmin, cmin] to the inst_info["box"] coordinates to get the right bbox coordinates.

So I changed this code (816 line at /infer /wsi.py -> inst_info["box"] += tissue_topleft) to " inst_info["box"] += tissue_topleft[::-1] ".

Can you check and fix it?

Thank you.
